### ISSUE TITLE
[BUG]: Testing on SN2 Benchmark runs into issue with scaler

### ISSUE THREAD

**Issue by am2145** [chemprop/chemprop#800]
**Describe the bug**
When running the benchmark using v2 on the SN2 reaction barrier height benchmark, I get very strange results when the output scaler is enabled. The MAE on the test set is extremely high at 330 kcal/mol despite low training losses and a validation error of 2.7 kcal/mol. When I disable the output scaler, the test set MAE now matches the validation MAE and the predictions look much more reasonable. I'm not sure what is occuring, but some scaling / unscaling seems to be going awry here.  

**Screenshots**
With output scaler:
![image](https://github.com/chemprop/chemprop/assets/79985824/239d67e0-42d0-4ef9-ba2d-f17c914d34f7)

Without output scaler:
![image](https://github.com/chemprop/chemprop/assets/79985824/94ee8ccf-3661-4a72-bb65-f5a3eda3097a)

**Comment by KnathanM** [comment]
Closed by #803
